synapse/config/server.py

```diff
@@ -384,6 +400,15 @@ def default_config(self, server_name, data_dir_path, **kwargs):
         #
         #restrict_public_rooms_to_local_users: true
 
+        # The default room version for newly created rooms.
+        #
+        # Known room versions are listed here:
+        # https://matrix.org/docs/spec/#complete-list-of-room-versions
+        #
+        # For example, for room version 1, default_room_version should be set
+        # to "1".
+        default_room_version: %(DEFAULT_ROOM_VERSION)s
```

Comment:
blank (or rather, `#`-only) line missing here

Reply:
suggest commenting this out by default, because (a) that's the convention (b) it means that we can change the default for people who don't deliberately override it.